codalab_service.py

```diff
@@ -586,18 +586,22 @@ def wait_rest_server(cmd):
             ]
             self.run_service_cmd(' && '.join(commands))
 
+            print_header('Initializing the database with alembic')
+            # We need to upgrade the current database revision to the most recent revision before any other database operations.
+            self.run_service_cmd(
+                'if [ $(alembic current | wc -l) -gt 0 ]; then echo upgrade; alembic upgrade head; fi'
+            )
+
             print_header('Creating root user')
             self.run_service_cmd(
                 wait_mysql(
                     'python3 scripts/create-root-user.py {}'.format(self.args.codalab_password)
                 )
             )
 
-            print_header('Initializing/migrating the database with alembic')
-            # The first time, we need to stamp; after that upgrade.
-            self.run_service_cmd(
-                'if [ $(alembic current | wc -l) -gt 0 ]; then echo upgrade; alembic upgrade head; else echo stamp; alembic stamp head; fi'
-            )
+            print_header('Stamping the database with alembic')
```

Comment:
This is only necessary when the `upgrade` condition is false, so maybe we want to add the if statement in there.